baselines/driving_smarts/benchmark.py

```diff
@@ -0,0 +1,140 @@
+# smarts/benchmark/driving_smarts_competition_benchmark.py
```

Comment:
Perhaps change comment as file does not exist.

Reply:
It is removed.